catalog/app/containers/Package/index.js

```diff
@@ -168,14 +170,18 @@ const Install = ({ name, owner }) => (
       <FormattedMessage {...strings.installThen} />
     </p>
     <Code>
-      <Unselectable>$ </Unselectable>quilt install {owner}/{name}
+      <Unselectable>$ </Unselectable>quilt install {makeHandle(owner, name)}
     </Code>
     <p><FormattedMessage {...strings.sell} /></p>
     <Help href={blogManage} />
     <h2><FormattedMessage {...strings.access} /></h2>
     <Tabs>
       <Tab label="Python">
-        <Code>from quilt.data.{owner} import {name}</Code>
+        {
+          config.team ?
+            <Code>from quilt.team.{config.team.name}.{owner} import {name}</Code>
```

Comment:
I think this is a "friendly" name - e.g., "Quilt Test", not "quilttest". We should have two different variables.

Reply:
oh yeah. what i'd like to do for now is treat `config.team.name` as the case-sensitive team id. that's how i use it right now. i can rename it to `team.id` if that's better. i don't use the friendly string anywhere. and of course i can fix `config.js.tmpl`. separate PR.